bin/extract_version_notes.py

```diff
@@ -0,0 +1,22 @@
+""" A Python module that will read the contents of CHANGELOG.md and extract
+    the release notes for the most recent version.
+
+    This module assumes:
+
+    * The top entry in CHANGELOG.md is that of the release to be published.
+    * Each entry is separated by a new line and starts with a line: "## vX.Y.Z"
+
+"""
```

Comment:
Not the prettiest way to do this, but I couldn't quite get something like `csplit` to behave as I wanted.

Reply:
I don't really have a better way.